platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/User.java

```diff
@@ -88,6 +85,9 @@
     @Column(name = "last_login")
     private ZonedDateTime lastLogin;
 
+    @OneToMany(mappedBy = "user")
+    private List<Participation> participations;
```

Comment:
Should probably be a `Set` as well, or is that list ordered by date or something?